sources/packages/api/src/common/loggable.ts

```diff
@@ -0,0 +1,21 @@
+import { LoggerService } from "../logger/logger.service";
+
+export interface Loggable {
+  logger(): LoggerService;
```

Comment:
Is it possible to define a property instead of a method for the interface?

Reply:
DONE